porsche-digital-lab-berlin-blockchain/catbox
# Testing

## What does this do?
Adds additional tests for the `ipfs`, `cryptography`, `blockchain` and `mixed` modules.

## How to test?
make sure that you have the contracts build directory linked in the root of the `catbox` directory. You can do this by doing the following (make sure that you have built the contracts in the `catbox-contracts` repo):
```bash
ln -s ../<path-to-repo/build/contracts>
```
## Still Missing
event parsing tests relating to blockchain module... perhaps this is not needed since it is tested in the `catbox-contracts` repo?
